### PR TITLE
[#165864957] Manual payment form - wrong font family and items alignment

### PR DESCRIPTION
**Pull request by Undermaken**
This PR aims to align the payment form with the font family used as default into the app (Titillium)
As requested it removes the left padding from the form's items

### Before
form_pre

### After
form_post

### CODE REVIEW

**Comment by digitalcitizenship** [comment]
## Affected stories

  * 🐞 [#165864957](https://www.pivotaltracker.com/story/show/165864957): Il carattere tipografico nel form "dati pagamento" non è Titillium

<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: ts/screens/wallet/payment/ManualDataInsertionScreen.tsx [hunk 1]
```diff
@@ -170,6 +178,7 @@ class ManualDataInsertionScreen extends React.Component<Props, State> {
                 />
               </Item>
               <Item
+                style={{ marginLeft: 0 }}
```
Comment (fpersico) [inline ts/screens/wallet/payment/ManualDataInsertionScreen.tsx]: Can you check the others inline style.

File: ts/screens/wallet/payment/ManualDataInsertionScreen.tsx [hunk 2]
```diff
@@ -150,6 +150,9 @@ class ManualDataInsertionScreen extends React.Component<Props, State> {
           <Content scrollEnabled={false}>
             <H1>{I18n.t("wallet.insertManually.title")}</H1>
             <Text>{I18n.t("wallet.insertManually.info")}</Text>
+            {/* Native Base Form has a padding by design. So if we want to remove the left padding
```
Comment (fpersico) [inline ts/screens/wallet/payment/ManualDataInsertionScreen.tsx]: Useless comment

Comment (fpersico) [reply ts/screens/wallet/payment/ManualDataInsertionScreen.tsx]: Please remove so i can merge

Comment (Undermaken) [reply ts/screens/wallet/payment/ManualDataInsertionScreen.tsx]: @fpersico update! thank you

**Pull request merged**